internal/winapi/process.go

```diff
@@ -0,0 +1,100 @@
+package winapi
+
+import "syscall"
+
+const PROC_THREAD_ATTRIBUTE_PARENT_PROCESS = 0x00020000
+
+const PROCESS_ALL_ACCESS uint32 = 2097151
+
+// typedef struct _PROC_THREAD_ATTRIBUTE {
+//     ULONG_PTR Attribute;
+//     SIZE_T Size;
+//     ULONG_PTR Value;
+// } PROC_THREAD_ATTRIBUTE, *PPROC_THREAD_ATTRIBUTE;
+type PROC_THREAD_ATTRIBUTE struct {
+	Attribute *uint32
+	Size      uintptr
+	Value     uintptr
+}
+
+// typedef struct _PROC_THREAD_ATTRIBUTE_LIST {
+//     ULONG PresentFlags;
+//     ULONG AttributeCount;
+//     ULONG LastAttribute;
+//     ULONG SpareUlong0;
+//     PPROC_THREAD_ATTRIBUTE ExtendedFlagsAttribute;
+//     PROC_THREAD_ATTRIBUTE Attributes[1];
+// } PROC_THREAD_ATTRIBUTE_LIST;
+type PROC_THREAD_ATTRIBUTE_LIST struct {
+	PresentFlags           uint32
+	AttributeCount         uint32
+	LastAttribute          uint32
+	SpareUlong0            uint32
+	ExtendedFlagsAttribute *PROC_THREAD_ATTRIBUTE
+	Attributes             [1]*PROC_THREAD_ATTRIBUTE
```

Comment:
I think this should be `Attributes [1]PROC_THREAD_ATTRIBUTE` (no pointer)?

Reply:
I think I tried this and it didn't work out. Complained on the second InitializeProcThreadAttributeList call but let me try one more time.

Reply:
No I'm thinking of the pointer to this in STARTUPINFOEX actually. The above not being pointers works out fine